### DISCUSSION TITLE
Unable to get Working Directory in Console Title since upgrade 16.9

### DISCUSSION THREAD

**Discussion by stevenke** [JanDeDobbeleer/oh-my-posh#3918] [Troubleshoot] [answered]
I was previously using the powershell module for Oh-My-Posh and recently upgraded to the winget installer version.  I'm running on Windows and using the Windows Terminal.

Before upgrade I used the following :
  "console_title": true,
  "console_title_style": "path"

After the upgrade the above didn't work so I tried:
  "console_title": true,
  "console_title_template": " {{ .Folder }} "

Using the above template only puts the folder of the working directory (understandably), not the entire directory.  So I tried several other template values...
  "console_title_template": " {{ .Location }} "
  "console_title_template": " {{ .Path }} "
  "console_title_template": " {{ path .Location }} "
  "console_title_template": " {{ path .Path }} "

I'm guessing I'm missing something simple, but I just haven't found it and am hoping someone can help.

Thanks,
SteveK

**Comment by JanDeDobbeleer** [comment, marked as answer]
`"console_title_template": "{{.Folder}}"` works and that indeed adds the current folder. If you want the full path, use `"console_title_template": "{{ .PWD }}"`. See [here](https://ohmyposh.dev/docs/configuration/title), this leverages the global template properties listed [here](https://ohmyposh.dev/docs/configuration/templates#global-properties).

**Reply by stevenke** [reply]
Thank you... I totally missed ".PWD" 😞
